openedx_lti_tool_plugin/middleware.py

```diff
@@ -76,5 +76,9 @@ def process_view(
         ):
             return None
 
-        log.error('LTI Middleware: User %s path request blocked: %s', request.user, request.path)
+        log.error(
+            'LTI Middleware: User %s path request blocked: %s',
+            request.user,
```

Comment:
Will this log the user's username, email or id? @kuipumu

Reply:
@Squirrel18 It will log user's username